python/triton/testing.py

```diff
@@ -17,7 +17,6 @@ def nvsmi(attrs):
 
 
 def do_bench_cudagraph(fn, rep=20, grad_to_none=None):
-    import torch
```

Comment:
A method's comment needs to be immediately after def for it to get picked up as its doc string by docs builder. Same for other method below.